tap_linkedin_ads/client.py

```diff
@@ -141,12 +149,46 @@ def __init__(self,
                           max_tries=5,
                           factor=2)
     def __enter__(self):
-        self.__verified = self.check_access_token()
+        self.__verified = self.get_access_token()
```

Comment:
I don't think `get_access_token` returns anything here, so I don't think `self.__verified` will ever be set to anything.

I also don't think we use `check_access_token` anymore. Do we still need that function?